client/js/store/map/ProcedureMapSettings.js

```diff
@@ -39,9 +39,10 @@ export default {
         .then(response => checkResponse(response))
     },
 
-    fetchProcedureMapSettings ({ commit }, procedureId) {
+    fetchProcedureMapSettings ({ commit }, args= { procedureId, isMaster }) {
```

Comment:
this looks un-javascript
```suggestion
    fetchProcedureMapSettings ({ commit }, { procedureId, isMaster }) {
```

Reply:
This does not work. `isMaster` is always undefined this way.

Reply:
Thats weired. But procedureId is set proppery?

Reply:
maybe then: 
```suggestion
    fetchProcedureMapSettings ({ commit }, args= { procedureId, isMaster = false }) {
```
?

I really don't understand, why it doesn't work as expected :-/

Reply:
> Thats weired. But procedureId is set proppery?

Yes.

> I really don't understand, why it doesn't work as expected :-/

I'm not sure. I think it has something to do with how we call it? I tried several ways, this was the only way I could make it work.

Reply:
Added space here [621a77c](https://github.com/demos-europe/demosplan-core/pull/3738/commits/621a77c837fdb7c5ad735e6ab1e590658b1c1b39)

Reply:
I have no idea what I did wrong before, but now it seems to work like expected without the `args`. Done here [ecf51f2](https://github.com/demos-europe/demosplan-core/pull/3738/commits/ecf51f259e7f05b7e00021b5d6accb312866f014) Sorry for the confusion.